soh/soh/Enhancements/gameplaystats.cpp

```diff
@@ -664,7 +666,7 @@ void SetupDisplayColors() {
     }
 }
 
-void InitStatTracker() {
+extern "C" void InitStatTracker() {
```

Comment:
This feels funky to me. I wonder if we wrap the header declaration of this method with `#ifdef __cplusplus`, would that allow us to remove the `extern "C"` here, since `InitStatTracker()` is only called in a cpp file.

Reply:
I can say that it does compile and run with your suggestion, but I don't know the relative positives/negatives for each method.

Reply:
Not wrapping it allows this code to be called in C as well as C++. The fix in these cases is to link it as C rather than link it as C++ and only include it for C++ code.

Reply:
So is this fine how it is?

Reply:
Yes